src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.CreateFile.cs

```diff
@@ -33,8 +35,9 @@ internal partial class Kernel32
             int dwFlagsAndAttributes,
             IntPtr hTemplateFile)
         {
-            lpFileName = PathInternal.EnsureExtendedPrefixOverMaxPath(lpFileName);
-            return CreateFilePrivate(lpFileName, dwDesiredAccess, dwShareMode, ref securityAttrs, dwCreationDisposition, dwFlagsAndAttributes, hTemplateFile);
+            string? lpFileNameWithPrefix = PathInternal.EnsureExtendedPrefixOverMaxPath(lpFileName); // TODO-NULLABLE: API marked as `DO NOT USE`
```

Comment:
This isn't a nullable-related todo.

Reply:
@JeremyKuhne, we're still using APIs we're not supposed to be using apparently?  Can we either fix the comments on the target method or update the code to not use that method and delete it?

Reply:
Yes, it needs removed, it appears to have gotten lost in the shuffle. The only ones left are here and FindFirstFileEx that I can see.

Reply:
replacing with `EnsureExtendedPrefixIfNeeded` (per comment) and removing API marked as `DO NOT USE`